src/components/OutputsTable/VariantEditor.tsx

```diff
@@ -22,7 +22,13 @@ export default function VariantEditor(props: { variant: PromptVariant }) {
     setIsChanged(currentFn.length > 0 && currentFn !== lastSavedFn);
   }, [lastSavedFn]);
 
-  useEffect(checkForChanges, [checkForChanges, lastSavedFn]);
```

Comment:
@corbt Was this important? I can see it being beneficial if multiple people are working on the same prompt (though even then I think I'd prefer to update the user's editor to match the server so they don't accidentally overwrite their collaborator's saved work)

Reply:
This is to show/hide the "save" button depending on whether your editor matches the last saved state. If you remove it I don't think the "save" button will show/hide reliably?